utils/luci-app-owm/luasrc/owm.lua

```diff
@@ -285,14 +285,19 @@ function get()
 		revision=version.distversion --owm
 	}
 
+
+	local email2owm = uci:get_first("freifunk", "owm", "submit_email")
+	if not email2owm then
+		email2owm = '0'
+	end
```

Comment:
i think we can shorten these 4 lines to `local submit_email = tonumber(uci:get_first("freifunk", "owm", "submit_email")) or 0`